### ISSUE TITLE
[rocm-comgr] Linking fails during build

### ISSUE THREAD

**Issue by tpkessler** [rocm-arch/rocm-arch#113]
When compiling `rocm-comgr`, the following error occurs:
```
[ 55%] Linking CXX shared library libamd_comgr.so
/usr/bin/ld: CMakeFiles/amd_comgr.dir/src/comgr-compiler.cpp.o: in function `COMGR::linkWithLLD(llvm::ArrayRef<char const*>, llvm::raw_ostream&, llvm::raw_ostream&)':
comgr-compiler.cpp:(.text+0x5ee7): undefined reference to `lld::elf::link(llvm::ArrayRef<char const*>, bool, llvm::raw_ostream&, llvm::raw_ostream&)'
```
Albeit the function is present in `/opt/rocm/lib`:

```
$ nm -C /opt/rocm/lib/liblldELF.a| grep lld::elf::link
0000000000000000 T lld::elf::link(llvm::ArrayRef<char const*>, bool, llvm::raw_ostream&, llvm::raw_ostream&)
```

**Comment by tpkessler** [comment]
Fixed with PR #114

**Comment by guiguid** [comment]
fixed with : 
`-DCMAKE_EXE_LINKER_FLAGS="-Wl,-rpath-link,/opt/rocm/lib" \`
only
